apps/omg_watcher_info/test/support/db_case.ex

```diff
@@ -0,0 +1,64 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.WatcherInfo.DBCase do
```

Comment:
❤️

Reply:
I tried, but I spent too much time fixing stuff, so I removed it for now. I think this needs to be refactored first: https://github.com/omgnetwork/elixir-omg/blob/master/apps/omg_watcher_info/lib/omg_watcher_info/supervisor.ex#L24

Reply:
maybe this would help? https://github.com/omgnetwork/childchain/blob/b2ecb0c2ac4e554e95b9a04261d983cad848c2fa/apps/engine/test/support/db/data_case.ex

Reply:
Ahaha okay then we're on par. Last time I tried I got stuck on the supervisor as well,

The issue is that we have to remove `:phoenix_ecto_sandbox` completely from all tests in one go, otherwise we have the common way (that Ino mentioned) and our fixture way, and the test suite break when there're two ways of sandboxing at the same time.

Reply:
Yes, exactly!
I'm planning to make a PR for that in the future